src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -137,15 +139,28 @@ Greeting.propTypes = {
 };
 
 export default function WebexActivityStream(props) {
-  const {roomID, adapter} = props;
-  const {title, roomType} = useRoom(roomID, adapter);
-  const activityIDs = useActivityStream(roomID, adapter);
+  const {roomID, adapters} = props;
+  const {roomsAdapter, activitiesAdapter, peopleAdapter} = adapters;
+  const {title, roomType} = useRoom(roomID, roomsAdapter);
+  const activityIDs = useActivityStream(roomID, roomsAdapter);
   const personName = roomType === RoomType.DIRECT ? title : '';
+  const activities = activityIDs.map((activityID) => (
+    <WebexActivity key={activityID} activityID={activityID} adapters={{activitiesAdapter, peopleAdapter}} />
+  ));
 
-  return <div className="activity-stream">{!activityIDs.length && <Greeting personName={personName} />}</div>;
+  return (
+    <div className="activity-stream">
+      {!activityIDs.length && <Greeting personName={personName} />}
+      <div>{activities}</div>
```

Comment:
This should be a conditional, e.g. either display the greeting or display activities
Otherwise the activities div will always show

Reply:
yeah I will take care of it